src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -46,16 +59,28 @@ public class ErrorController implements FxController {
 	private final Stage window;
 	private final Environment environment;
 
-	private BooleanProperty copiedDetails = new SimpleBooleanProperty();
+	private final BooleanProperty copiedDetails = new SimpleBooleanProperty();
+	private final BooleanProperty errorSolutionFound = new SimpleBooleanProperty();
```

Comment:
This doesn't need to be a `BooleanProperty` that you explicitly need to set. Instead it can be derived from `matchingErrorDiscussion`:

```suggestion
	private final ObjectProperty matchingErrorDiscussion = new SimpleObjectProperty();
	private final BooleanExpression errorSolutionFound = matchingErrorDiscussion.isNotNull();
```

Reply:
modified to ObjectProperty and BooleanExpression